### PR TITLE
Some typos and conventions fixed 

### PR DESCRIPTION
**Pull request by instruccionesaldorso**

### CODE REVIEW

File: Documentation/5-Domain/1-the-application-domain.rst [hunk 1]
```diff
@@ -16,18 +16,18 @@ following characteristics were defined:
...
 * Every member organization can add, edit and delete their offers.
-  The offers can searched by the users and filterd by several criteria. Filter criteria are specially:
+  The offers can be searched by the users and filtered by several criteria. Filter criteria are specially:
 
-  the duration of validity of the offer
+  - the duration of validity of the offer
 
-  the target audience for which the offer is straightened (mostly
+  - the target audience for which the offer is straightened (mostly
   the age span, but also physical handicapped, swimmer and so on)
```
Comment (DanielSiepmann) [inline Documentation/5-Domain/1-the-application-domain.rst]: The indentation is slightly wrong, this way the rendering would probably break. Following line breaks have to be indented in the same level as the text above.

Same goes for the following list items, prefixed with `-`. If you will not fix that, we will fix it afterwards.

**Comment by DanielSiepmann** [comment]
Thanks for all the effort @instruccionesaldorso.

File: Documentation/5-Domain/1-the-application-domain.rst (hunk 1 above)
Comment (instruccionesaldorso) [reply Documentation/5-Domain/1-the-application-domain.rst]: Would you mind fix it afterwards, @DanielSiepmann ?

Comment (DanielSiepmann) [reply Documentation/5-Domain/1-the-application-domain.rst]: Of course. I already merged and will fix it now. Thanks for the contribution again.

**Pull request merged**